strawberryfields/apps/graph/utils.py

```diff
@@ -153,3 +153,76 @@ def is_subgraph(subgraph: Iterable, graph: nx.Graph):
         return set(subgraph).issubset(graph.nodes)
     except TypeError:
         raise TypeError("subgraph and graph.nodes must be iterable")
+
+
+def is_clique(graph: nx.Graph) -> bool:
+    """Determines if the input graph is a clique
+
+    Args:
+        graph (nx.Graph): the input graph
+
+    Returns:
+        bool: returns ``True`` if input graph is a clique and ``False`` otherwise
+    """
+    edges = graph.edges
+    nodes = graph.order()
+
+    return len(edges) == nodes * (nodes - 1) / 2  # a clique has of n nodes has n*(n-1)/2 edges
+
+
+def c_0(subgraph: list, graph: nx.Graph):
```

Comment:
This is very minor, but since the name is defined in the paper as C0, should the function be `c0(subgraph, graph)`?

Reply:
Note that `c0_nodes` is used within the function, not `c_0_nodes`.

Reply:
After going through the rest of the PR, everything else uses the `c0` (no underscore) convention, i.e., `test_c0_comp_graph`

Reply:
lol I didn't see your review before I made mine and came to the same conclusion :laughing:

Reply:
As long as it is consistent throughout code/docs, I have no issues with what you decide